src/ResourceManager/SiteRecovery/Commands.SiteRecovery/Fabrics/GetAzureRmSiteRecoveryFabric.cs

```diff
@@ -29,6 +29,9 @@ namespace Microsoft.Azure.Commands.SiteRecovery
     /// </summary>
     [Cmdlet(VerbsCommon.Get, "AzureRmSiteRecoveryFabric", DefaultParameterSetName = ASRParameterSets.Default)]
     [OutputType(typeof(List<ASRFabric>))]
+    [Obsolete("This cmdlet has been marked for deprecation in an upcoming release. Please use the " +
```

Comment:
@viverm for all of these `Obsolete` attributes, I would point at the `upcoming-breaking-changes.md` file with the cmdlet mapping table to make it easier for users to identify which cmdlets to use in the new module